library/L2_HAL/displays/lcd/st7066u.hpp

```diff
@@ -247,28 +213,28 @@ class St7066u
 
   /// Clears all characters on the display by sending the clear display command
   /// to the device.
-  virtual void ClearDisplay()
+  void ClearDisplay() const
   {
     WriteCommand(Command::kClearDisplay);
     sjsu::Delay(2ms);  // Clear display operation requires 1.52ms
```

Comment:
```suggestion
    sjsu::Delay(1.6ms);  // Clear display operation requires 1.52ms
```
You can do that now. But if you want to be safe, keeping it 2ms is also fine.